src/layer/vector/SVG.js

```diff
@@ -148,6 +144,12 @@ export const SVG = Renderer.extend({
 		} else {
 			path.setAttribute('fill', 'none');
 		}
+
+		if (options.interactive) {
+			path.classList.add('leaflet-interactive');
```

Comment:
Should we not also be calling `addInteractiveTarget()` and `removeInteractiveTarget()` here like we do in `DivOverlay`?

Reply:
Makes sense! Pushed a change :)